### PR TITLE
adds ticket 5

### PR DESCRIPTION
**Pull request by Vondry66**

### CODE REVIEW

**Review by Augs0 — CHANGES_REQUESTED** [review]
Hi Filip, the code all looks good but there's an error with one of your imports. Your utils function doesn't seem to be being imported properly so when I run your app I get 'Error: Can't resolve '../utils/api'' in both Category and Reviews. Please fix this error and resubmit your PR

File: src/App.js [hunk 1]
```diff
@@ -4,14 +4,16 @@ import Reviews from "./components/Reviews";
...
+
 class App extends Component {
   render() {
     return (
       <div className="App">
         <Header />
         <Nav />
         <Routes>
-          <Route path="Reviews" element={<Reviews />}>r</Route>
+          <Route path="Reviews" element={<Reviews />}></Route>
```
Comment (mallionaire) [inline src/App.js]: Watch your casing and paths. 
should be lowercase and start with a `/` eg `/reviews`

File: src/components/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useEffect,useState } from "react";
+import { Link } from "react-router-dom";
+import { getCategory } from "../utils/api";
+
+const Category =()=>{
+    const[categories,setCategories]=useState([])
+    useEffect(()=>{
+        getCategory().then((category)=>{
+            setCategories(category)
+        }).catch(err=>{})
```
Comment (mallionaire) [inline src/components/Category.jsx:10]: This `.catch` is an empty code block.

File: src/components/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },[])
+    return(<main>
+         <h2 className="h2">Read reviews by category</h2><ul>
+        {categories.map((cat)=>{
+            
+            
+           return<li><Link to={`/reviews/${cat.slug}`}>{cat.slug}</Link></li>
```
Comment (mallionaire) [inline src/components/Category.jsx:17]: Read your console errors - `each child in a list...`

File: src/components/Nav.jsx [hunk 4]
```diff
@@ -2,8 +2,8 @@ import { Link } from "react-router-dom";
 
 const Nav = ()=>{
     return ( <div>    <h1>Are you "Board"?</h1>
-    <Link to="/Reviews">Reviews</Link>
-   
+    <Link to="/Reviews">Reviews   </Link>
```
Comment (mallionaire) [inline src/components/Nav.jsx]: see comment above about casing

File: src/utils/api.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
```
Comment (mallionaire) [inline src/utils/api.js]: uneccessary imports

File: src/utils/api.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      params: { category: categories_slug },
+    })
+    .then((res) => {
+      return res.data;
+    });
+};
+
+export const getCategory = () => {
+  return gamesApi.get("/categories").then((res) => {
+    console.log(res.data.categories);
```
Comment (mallionaire) [inline src/utils/api.js]: remove console.logs please

**Review by mallionaire — CHANGES_REQUESTED** [review]

**Pull request merged**